src/main/java/com/ir/example/miniticket/dao/command/InsertUserCommand.java

```diff
@@ -0,0 +1,36 @@
+package com.ir.example.miniticket.dao.command;
+
+import com.ir.example.miniticket.model.User;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+
+import java.sql.Date;
+import java.util.function.Consumer;
+
+public class InsertUserCommand implements Consumer<User> {
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+    private String INSERT = "insert into users (id,name,cpf,mobile_phone,email,password,birthday_date,gender,address) " +
+                            "values (:id,:name,:cpf,:mobile_phone,:email,:password,:birthday_date,:gender,:address)";
+
+    public InsertUserCommand(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = new NamedParameterJdbcTemplate(jdbcTemplate);
+    }
+
+    @Override
+    public void accept(User user) {
+        MapSqlParameterSource parameters = new MapSqlParameterSource();
+        parameters.addValue("id", user.id());
+        parameters.addValue("name", user.name());
+        parameters.addValue("cpf", user.cpf());
+        parameters.addValue("mobile_phone", user.mobilePhone());
+        parameters.addValue("email", user.email());
+        parameters.addValue("password", user.password());
```

Comment:
I know this is not the goal of this exercise. But to add information here, we generally don't save passwords in plain text like this. We generally apply a encryption algorithm to it, here is a reference that explains that much better than me: https://docs.oracle.com/cd/E26180_01/Platform.94/ATGPersProgGuide/html/s0506passwordhashing01.html

Reply:
For sure! In a real project I never would use plain text for password. Is it necessary to fix this in this project?